packages/contracts/npm/artifacts.ts

```diff
@@ -4,26 +4,23 @@
 import * as PermissionLib from '../artifacts/contracts/core/permission/PermissionLib.sol/PermissionLib.json';
 import * as IPermissionCondition from '../artifacts/contracts/core/permission/IPermissionCondition.sol/IPermissionCondition.json';
-import * as DAO from '../artifacts/contracts/core/DAO.sol/DAO.json';
-import * as IDAO from '../artifacts/contracts/core/IDAO.sol/IDAO.json';
-import * as DaoAuthorizable from '../artifacts/contracts/core/component/dao-authorizable/DaoAuthorizable.sol/DaoAuthorizable.json';
+import * as DAO from '../artifacts/contracts/core/dao/DAO.sol/DAO.json';
+import * as IDAO from '../artifacts/contracts/core/dao/IDAO.sol/IDAO.json';
+import * as DaoAuthorizable from '../artifacts/contracts/core/plugin/dao-authorizable/DaoAuthorizable.sol/DaoAuthorizable.json';
 
-// Factories
-import * as DAOFactory from '../artifacts/contracts/factory/DAOFactory.sol/DAOFactory.json';
-import * as TokenFactory from '../artifacts/contracts/factory/TokenFactory.sol/TokenFactory.json';
+// Framework contracts
+import * as DAOFactory from '../artifacts/contracts/framework/dao/DAOFactory.sol/DAOFactory.json';
+import * as TokenFactory from '../artifacts/contracts/framework/utils/TokenFactory.sol/TokenFactory.json';
+import * as DAORegistry from '../artifacts/contracts/framework/dao/DAORegistry.sol/DAORegistry.json';
 
-// Registry
-import * as DAORegistry from '../artifacts/contracts/registry/DAORegistry.sol/DAORegistry.json';
+// Plugin Contracts
+import * as TokenVoting from '../artifacts/contracts/plugins/governance/majority-voting/token/TokenVoting.sol/TokenVoting.json';
+import * as AddresslistVoting from '../artifacts/contracts/plugins/governance/majority-voting/addresslist/AddresslistVoting.sol/AddresslistVoting.json';
+import * as MerkleMinter from '../artifacts/contracts/plugins/token/creation/MerkleMinter.sol/MerkleMinter.json';
+import * as MerkleDistributor from '../artifacts/contracts/plugins/token/distribution/MerkleDistributor.sol/MerkleDistributor.json';
 
-// Tokens
-import * as GovernanceERC20 from '../artifacts/contracts/tokens/GovernanceERC20.sol/GovernanceERC20.json';
-import * as GovernanceWrappedERC20 from '../artifacts/contracts/tokens/GovernanceWrappedERC20.sol/GovernanceWrappedERC20.json';
-
-// DAO Plugins
-import * as TokenVoting from '../artifacts/contracts/voting/token/TokenVoting.sol/TokenVoting.json';
-import * as AddresslistVoting from '../artifacts/contracts/voting/addresslist/AddresslistVoting.sol/AddresslistVoting.json';
-import * as MerkleDistributor from '../artifacts/contracts/tokens/MerkleDistributor.sol/MerkleDistributor.json';
-import * as MerkleMinter from '../artifacts/contracts/tokens/MerkleMinter.sol/MerkleMinter.json';
+import * as GovernanceERC20 from '../artifacts/contracts/plugins/token/governance/GovernanceERC20.sol/GovernanceERC20.json';
+import * as GovernanceWrappedERC20 from '../artifacts/contracts/plugins/token/governance/GovernanceWrappedERC20.sol/GovernanceWrappedERC20.json';
```

Comment:
Do we still have files named like GovernanceErc20?

Reply:
What are you asking? `...Erc20...` instead of `...ERC20...`?

What's your suggestion?